scripts/check.cmd

```diff
@@ -0,0 +1,4 @@
+isort .
```

Comment:
It seems this is specific to your setup (Windows). Feel free to add `scripts-private` to `.gitignore` and put this there.

Reply:
I thought it could be helpful for whomever is going to work on pysparkling on a Windows PC. It's making deployment a lot easier :-)

Reply:
Ok, moved it into a scripts_private directory now as requested. Works fine by me.